testapps/php56_extensions/composer.json

```diff
@@ -1,6 +1,33 @@
 {
     "require": {
         "php": "^5.6",
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "4.8.*",
```

Comment:
You mean if the developer puts a required version for the extension? Every example I've found of people using this field use "*" for the required version because extension versioning is inconsistent across extensions.